target/linux/ramips/image/mt76x8.mk

```diff
@@ -247,6 +247,15 @@ define Device/jotale_js76x8-32m
 endef
 TARGET_DEVICES += jotale_js76x8-32m
 
+define Device/kroks_kndrt31r19
+  IMAGE_SIZE := 15040k
+  DEVICE_VENDOR := Kroks
+  DEVICE_MODEL := Rt-Pot mXw DS RSIM
+  DEVICE_PACKAGES := kmod-usb2 luci-proto-qmi
```

Comment:
Well, we could swap it to just `uqmi` but the first time user experience would be a way worse..

Reply:
Do we also need `uboot-envtools` here?

Reply:
Please remove the dependency to `luci-proto-qmi` here. It should only depend on packages which are in the main OpenWrt repository `uqmi` should be fine.

Reply:
Done

Reply:
I removed the `luci-proto-qmi` dependency but for some reason still have the QMI interface setting available in LuCI:
![image](https://user-images.githubusercontent.com/233979/185965475-b9bf9526-a6fe-4171-a22b-d1a4761eb573.png)
![image](https://user-images.githubusercontent.com/233979/185965952-42f13880-0135-49d6-a5b8-fb14009d732b.png)
Any idea why?

Reply:
Oh, this qmi proto was cached in browser somehow!
No such thing in Incognito Mode:
![image](https://user-images.githubusercontent.com/233979/185970667-af38484c-fbf3-43c0-8d8c-c16621c8c94a.png)